app/kumactl/cmd/install/install_crds.go

```diff
@@ -0,0 +1,145 @@
+package install
+
+import (
+	"context"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kumahq/kuma/app/kumactl/pkg/install/k8s"
+	controlplane "github.com/kumahq/kuma/app/kumactl/pkg/install/k8s/control-plane"
+
+	k8s_apixv1beta1client "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/typed/apiextensions/v1beta1"
+
+	"github.com/kumahq/kuma/app/kumactl/pkg/install/data"
+)
+
+const KumaCrdApiGroup = "kuma.io"
+
+type InstallCrdsArgs struct {
+	NotInstalled bool
+}
+
+func newInstallCrdsCmd() *cobra.Command {
+	args := InstallCrdsArgs{}
+
+	cmd := &cobra.Command{
+		Use:   "crds",
+		Short: "Install Kuma Custom Resource Definitions on Kubernetes",
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			wantCrdFiles, err := GetCrdsFiles()
+			if err != nil {
+				return errors.Wrap(err, "Failed to read CRD files")
+			}
+
+			if !args.NotInstalled {
+				singleFile := data.JoinYAML(wantCrdFiles)
+
+				if _, err := cmd.OutOrStdout().Write(singleFile.Data); err != nil {
+					return errors.Wrap(err, "Failed to output rendered resources")
+				}
+
+				return nil
```

Comment:
might be nice to build the `singleFile` for the other case in another branch, and then only have one output? like `singleFile = args.NotInstalled ? buildNotInstalledFile() : data.JoinYaml(wantCrdFiles)` ?

Reply:
I did it this way to return quickly if there is no need to proceed if we just want all manifests

I'm gonna resolve this conversation as this PR is already too long open, but feel free to create a PR with this as an enhancement or poke me on our slack, and then we can figure this out together :-)

Reply:
Sounds good to me -- agreed, let's get this in!